ote/src/cljs/ote/views/service_search.cljs

```diff
@@ -241,13 +241,23 @@
 
       [operator-result-chips e! chip-results]]))
 
+(defn- capitalize-operation-area [sentence]
+  (str/replace sentence #"([^A-Öa-ä0-9_])(\w)"
```

Comment:
why are capital letters in a different range than lower case letters? what about Å?